x-pack/plugins/security_solution/public/flyout/entity_details/user_right/hooks/use_observed_user.ts

```diff
@@ -68,7 +58,7 @@ export const useObservedUser = (userName: string): ObserverUser => {
   return useMemo(
     () => ({
       details: observedUserDetails,
-      isLoading: loadingObservedUser,
+      isLoading: loadingObservedUser && loadingLastSeen && loadingFirstSeen,
```

Comment:
Shouldn't this be `||` instead? As in, it's loading if any of the sub-terms is also loading? 

Or is that not the intent here?